std_msgs/String frame_id
geometry_msgs/Point center
geometry_msgs/Vector3 dimensions
---
int64 num_points
